storage/testonly/admin_storage_tester.go

```diff
@@ -120,7 +120,7 @@ var (
 		HashStrategy:       trillian.HashStrategy_RFC6962_SHA256,
 		HashAlgorithm:      spb.DigitallySigned_SHA256,
 		SignatureAlgorithm: spb.DigitallySigned_ECDSA,
-		DisplayName:        "Pavel's Pre-ordered Log",
+		DisplayName:        "Pre-ordered Log",
```

Comment:
This line was longer than 20 chars, which was against the MySQL schema.

Reply:
We could increase that number, not sure why I made it so short.

Reply:
Meh, I'm fine not having my name in the code :)

Reply:
This is indeed wise.